### PR TITLE
SEO-176062-xamarin-UG-redirect

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: Xamarin/CheckBox/Visual-Customization.md [hunk 1]
```diff
@@ -183,4 +183,4 @@ stackLayout.Children.Add(sfCheckBox);
 
 ## See also
 
-[How to get the values of selected checkboxes in a group using Xamarin.Forms](https://www.syncfusion.com/kb/10915/how-to-get-the-values-of-selected-checkboxes-in-a-group-using-xamarin-forms)
+[How to get the values of selected checkboxes in a group using Xamarin.Forms](https://support.syncfusion.com/kb/article/9507/how-to-get-the-values-of-selected-checkboxes-in-a-group-using-xamarin-forms?isInternalRefresh=False)
```
Comment (Yvone-Atieno) [inline Xamarin/CheckBox/Visual-Customization.md:186]: Check KB links

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/CheckBox/Visual-Customization.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/CheckBox/Visual-Customization.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**